template/web/src/store/supportedChains.ts

```diff
@@ -2,23 +2,23 @@ import { baseSepolia, Chain, base } from 'viem/chains';
 import { Environment, getCurrentEnvironment } from './environment';
 
 // The list of supported Chains for a given environment
-export const SUPPORTED_CHAINS = new Map<Environment, Chain[]>([
-  [Environment.localhost, [baseSepolia]],
-  [Environment.development, [baseSepolia]],
-  [Environment.staging, [base]],
-  [Environment.production, [base]],
-]);
+export const SUPPORTED_CHAINS: Record<Environment, [Chain, ...Chain[]]> = {
+  [Environment.localhost]: [baseSepolia],
+  [Environment.development]: [baseSepolia],
+  [Environment.staging]: [base],
```

Comment:
should add `baseSepolia` here as well and also in prod because someone might want to deploy the app only to baseSepolia

Reply:
done 👍